src/test/java/seedu/address/testutil/EditPersonDescriptorBuilder.java

```diff
@@ -102,6 +104,16 @@ public EditPersonDescriptorBuilder withIncome(double income) {
         return this;
     }
 
+    /**
+     * Sets the {@code FamilySize} of the {@code EditPersonDescriptor} that we are building.
+     */
+    public EditPersonDescriptorBuilder withFamilySize(int familySize) {
+        assert FamilySize.isValidFamilySize(familySize);
```

Comment:
Is this assertion necessary if the `FamilySize` constructor already checks the validity of `familySize`?

Reply:
Agree, will remove.